pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -242,7 +317,31 @@ func (tc *testCase) prepareTestClient(t *testing.T) *fake.Clientset {
 	fakeWatch := watch.NewFake()
 	fakeClient.AddWatchReactor("*", core.DefaultWatchReactor(fakeWatch, nil))
 
-	return fakeClient
+	gv := unversioned.GroupVersion{Group: "extensions", Version: "v1beta1"}
+	scale := &fakeScale{
+		gv:   gv,
+		path: fmt.Sprintf("/apis/%s/%s/namespaces/%s/replicationcontrollers/%s/scale", gv.Group, gv.Version, namespace, rcName),
```

Comment:
This is not the new path we registered for replicationcontrollers. Is that deliberate?